terraform/modules/happy-stack-eks/variables.tf

```diff
@@ -50,6 +50,7 @@ variable "services" {
     port : optional(number, 80),
     scheme : optional(string, "HTTP"),
     service_port : optional(number, null),
+    service_scheme : optional(string, "HTTP"),
```

Comment:
We need a validation to ensure this scheme is either `HTTP` or `HTTPS`

Reply:
Thanks Alex, I added validations for the three scheme fields.